### PR TITLE
crazycat: fix building for Amlogic devices using 3.10 kernel and enable for WeTek_Core

### PR DESCRIPTION
**Pull request by kszaq**
Recently introduced patches to include drivers for internal DVB modules can't be used with 3.10 devices. Fix this by providing separate patch directory.

### CODE REVIEW

File: packages/linux-driver-addons/dvb/crazycat/package.mk [hunk 1]
```diff
@@ -35,8 +35,10 @@ PKG_ADDON_NAME="DVB drivers for TBS (CrazyCat)"
 PKG_ADDON_TYPE="xbmc.service"
 PKG_ADDON_VERSION="${ADDON_VERSION}.${PKG_REV}"
 
-if [ $LINUX = "amlogic-3.14" -o $LINUX = "amlogic-3.10" ]; then
-  PKG_PATCH_DIRS="amlogic"
+if [ $LINUX = "amlogic-3.10" ]; then
+  PKG_PATCH_DIRS="amlogic-3.10"
+elif [ $LINUX = "amlogic-3.14" ]; then
+  PKG_PATCH_DIRS="amlogic-3.14"
```
[the remark this answers is not in the document]
Comment (CvH) [reply packages/linux-driver-addons/dvb/crazycat/package.mk]: I think we should rename it `amlogic-$version`, `amlogic` is a bit misleading and is not needed at all (hopefully) *if* we get mainline linux support some day. 
Ofc it basically does not matter if we do this this or that way, but as this patches are heavily depending at the used kernel, it isn't wrong.

**Comment by CvH** [comment]
it looks okay, I have no hw so I can't test it runtime at all (@codesnake should be able to)

**Comment by kszaq** [comment]
@CvH This is runtime-tested on S805 device. As it's not in LE master, I asked @codesnake for review using W Core.

**Comment by chewitt** [comment]
PR's for `master` should not be tagged for the `amlogic` dev branch please.

**Comment by codesnake** [comment]
@kszaq To make the above adapters work please enable CONFIG_I2C_MUX=y in WeTek Core kernel config.

**Comment by kszaq** [comment]
@codesnake Thank you for your review. I have added a kernel update patch, please check if it's OK.

**Pull request merged**